lib/mods/nom_terminology.rb

```diff
@@ -3,10 +3,7 @@ module Mods
   # from:  http://www.loc.gov/standards/mods/v3/mods-userguide-generalapp.html
 
   # Nokogiri 1.6.6 introduced lang as a built-in attribute
-  LANG_ATTRIBS = (Nokogiri::VERSION < "1.6.6") ?
```

Comment:
Should we drop the comment on line 5 as well, since `lang` is no longer a referenced attr?

Reply:
I think the comment is helpful at explaining why we don't have `lang` listed in the `LANG_ATTRIBUTES`

Reply:
OK, I buy it.